met-web/src/components/engagement/view/EngagementView.tsx

```diff
@@ -56,7 +56,11 @@ export const EngagementView = () => {
         if (!isPreview) {
             setDefaultPanel('email');
             setEmailModalOpen(true);
-            window.snowplow('trackPageView', 'Verify Email Modal');
+            try {
+                window.snowplow('trackPageView', 'Verify Email Modal');
+            } catch (error) {
+                console.log(error);
```

Comment:
You could add some more context for this particular snowplow code with:

```suggestion
                console.log('Verify email modal snowplow error:', error);
```